src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -194,6 +194,11 @@ public void changeAction(String action) {
     write(representation, LoanProperties.ACTION, action);
   }
 
+  public Loan withAction(String action) {
```

Comment:
Does it make sense for this method to accept LoanAction instead of a String as a parameter?

Reply:
I think so.